packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -1276,8 +1276,8 @@ export interface LanguagesMain {
     $registerQuickFixProvider(handle: number, pluginInfo: PluginInfo, selector: SerializedDocumentFilter[], codeActionKinds?: string[]): void;
     $clearDiagnostics(id: string): void;
     $changeDiagnostics(id: string, delta: [string, MarkerData[]][]): void;
-    $registerDocumentFormattingSupport(handle: number, pluginInfo: PluginInfo, selector: SerializedDocumentFilter[]): void;
-    $registerRangeFormattingProvider(handle: number, pluginInfo: PluginInfo, selector: SerializedDocumentFilter[]): void;
```

Comment:
Why renaming these is needed?
Why not keep it in sync with the [corresponding VS Code interface](https://github.com/microsoft/vscode/blob/e4256dd1a02339bb1d56647ecd9134bf38bc7c03/src/vs/workbench/api/common/extHost.protocol.ts#L381-L382).

Reply:
> When I run Theia with the following plugins:
> 
> * https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-java-debug/vscode-java-debug-0.26.0.vsix
> * https://download.jboss.org/jbosstools/static/jdt.ls/stable/java-0.63.0-2222.vsix
> * https://download.jboss.org/jbosstools/vscode/stable/vscode-quarkus/vscode-quarkus-1.5.0-324.vsix
> 
> I've got
> ![image](https://user-images.githubusercontent.com/1636395/92281295-f1451c00-ef03-11ea-891a-275634121109.png)
> 
> ```
> root ERROR [hosted-plugin: 95750] Promise rejection not handled in one second: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type object , reason: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type object
> root ERROR [hosted-plugin: 95750] With stack trace: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type object
>     at validateString (internal/validators.js:112:11)
>     at Object.join (path.js:1039:7)
>     at /tmp/vscode-unpacked/java-0.63.0-2222.vsix/extension/dist/extension.js:7:10673
>     at new Promise (<anonymous>)
>     at /tmp/vscode-unpacked/java-0.63.0-2222.vsix/extension/dist/extension.js:7:10643
>     at Object.<anonymous> (/tmp/vscode-unpacked/java-0.63.0-2222.vsix/extension/dist/extension.js:7:11385)
>     at Generator.next (<anonymous>)
>     at o (/tmp/vscode-unpacked/java-0.63.0-2222.vsix/extension/dist/extension.js:7:8792)
> ```

How do you run Theia? Did you try to run the test script?

Reply:
Aligned with VS Code, thanks!